frontend/app/page.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+import { ApolloClient, InMemoryCache, ApolloProvider } from "@apollo/client";
+import TableSwitcher from "@/components/TableSwitcher";
+
+const { BACKEND_URL } = process.env;
+
+const client = new ApolloClient({
+  uri: `http://localhost:4000/graphql`,
```

Comment:
Would localhost work in production? What would be a better way so it works in multiple environments?

Reply:
Yes, it would be better to fetch it from the environment.